Dockerfile.prod

```diff
@@ -0,0 +1,46 @@
+FROM php:7.3-apache
+
+ARG GH_SECURED_TOKEN
+ENV GH_SECURED_TOKEN $GH_SECURED_TOKEN
+
+ARG GH_TOKEN
+ENV GH_TOKEN $GH_TOKEN
+
+ARG SYMFONY_PHPUNIT_VERSION 
+ENV SYMFONY_PHPUNIT_VERSION $SYMFONY_PHPUNIT_VERSION
+
+ARG APP_ENV
+ENV APP_ENV $APP_ENV
+
+RUN apt-get update
+RUN apt-get install -y zip libzip-dev \
+	unzip \
+	vim \
+	sudo \
```

Comment:
Is this part _really_ necessary (though it's used later in the dockerfile) ? 🤔

Reply:
zip and unzip packages are necessary to speed up Composer steps, otherwise I removed unecessary ones